windows/ReactTestApp/MainPage.cpp

```diff
@@ -45,7 +45,20 @@ namespace winrt::ReactTestApp::implementation
 
             // If only one component is present load it automatically
             if (components.size() == 1) {
-                ReactRootView().ComponentName(to_hstring(components.at(0).appKey));
+                auto firstComponent = components.at(0);
+                ReactRootView().ComponentName(to_hstring(firstComponent.appKey));
+                if (firstComponent.initialProperties.has_value()) {
+                    ReactRootView().InitialProps(
+                        [firstComponent](winrt::Microsoft::ReactNative::IJSValueWriter writer) {
+                            writer.WriteObjectBegin();
+                            for (auto &&property : firstComponent.initialProperties.value()) {
+                                std::any value = property.second;
```

Comment:
We should avoid a copy here:
```suggestion
                                auto &value = property.second;
```